lib/pi_piper/sysfs/version.rb

```diff
@@ -0,0 +1,6 @@
+module PiPiper
+  class Driver; end
```

Comment:
becareful to the way gemspec require the version constant.

El Matou

On Tue, Mar 15, 2016 at 8:59 PM, Zshawn Syed <notifications@github.com>
wrote:

> In lib/pi_piper/sysfs/version.rb
> <https://github.com/PiPiper/sysfs/pull/1#discussion_r56230151>:
>
> > @@ -0,0 +1,6 @@
> > +module PiPiper
> > +  class Driver; end
>
> I think if you put require 'pi_piper' in this file: lib/pi_piper/sysfs.rb.
> Then it should work because that file will always be loaded before the
> files within lib/sysfs/*
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/PiPiper/sysfs/pull/1/files/8e26d6f04c610c8c4e1d2af980cf8b8fba6c6533#r56230151>
>